TezosKit/Conseil/GetSentTransactionsRPC.swift

```diff
@@ -24,15 +22,19 @@ public class GetSentTransactionsRPC: RPC<[Transaction]> {
       Header.contentTypeApplicationJSON,
       Header(field: "apiKey", value: apiKey)
     ]
-    let payload = """
-      {"fields": [],"predicates": [{"field": "kind","set": ["transaction"],"operation": "eq","inverse": false}, {"field": "source","set": ["\(account)"],"operation": "eq","inverse": false}],"orderBy": [{"field": "timestamp","direction": "desc"}],"limit": \(limit)}
-    """
+
+    let predicates: [ConseilPredicate] = [
+      ConseilQuery.Predicates.predicateWith(field: "kind", set: ["transaction"]),
+      ConseilQuery.Predicates.predicateWith(field: "source", set: [account])
+    ]
+    let orderBy: ConseilOrderBy = ConseilQuery.OrderBy.orderBy(field: "timestamp")
+    let payload: [String: Any] = ConseilQuery.query(predicates: predicates, orderBy: orderBy, limit: limit)
```

Comment:
For the three variables above you don't need to specify the type, the compiler will infer it